modules/ROOT/pages/install-manual.adoc

```diff
@@ -146,6 +146,9 @@ mkdir -p /opt/anypoint/runtimefabric && cp ./rtf-install-scripts/scripts/init.sh
 
 . Run the `init.sh` script in privileged mode on the controller VM.
 +
+[NOTE]
+You must have root privileges to run the `init.sh` script.
```

Comment:
Should the note come after the command line, so as not to separate it from its intro?